recipes/libnvshmem/build-libnvshmem.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -e
+
+mkdir -p $PREFIX/lib/cmake/nvshmem/
```

Comment:
```suggestion
set -e

check-glibc bin/* lib/*

mkdir -p $PREFIX/lib/cmake/nvshmem/
```
https://github.com/conda-forge/cf-nvidia-tools-feedstock/tree/main/recipe

Reply:
Done. Thanks!